### PR TITLE
DEV22 release

### PR DESCRIPTION
**Pull request by cmutel**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,34 +1,36 @@
...
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 41188a5ed32908ed6a442418689918c988ba289deede118d238f0e79cd52dabb
+  sha256: 210cd87819a9d469b3d842b9c8ca72db196ace049693901af6ef77c5b7d5528f
 
 build:
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
-  number: 0
+  number: 1
```
Comment (m-rossi) [inline recipe/meta.yaml:15]: A build number bump is not necessary as you increased the version.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,34 +1,36 @@
...
 
 requirements:
   host:
-    - python >=3.4
+    - python >=3.9
     - pip
     - setuptools
   run:
     - bw2parameters
+    - bw2calc >=2.0.dev13
```
Comment (m-rossi) [inline recipe/meta.yaml:26]: To make the package use another pre-release verison you have to follow the guide at https://conda-forge.org/docs/maintainer/knowledge_base.html#pre-release-builds

**Review by m-rossi — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (cmutel) [reply recipe/meta.yaml:15]: Sorry, instinct kicked in as I was revising the PR 😛

File: recipe/meta.yaml (hunk 2 above)
Comment (cmutel) [reply recipe/meta.yaml:26]: Thanks! Actually, we will just drop this instead: https://github.com/brightway-lca/brightway2-data/issues/145

**Pull request closed without merging**